plotly/offline/offline.py

```diff
@@ -335,7 +335,33 @@ def iplot(figure_or_data, show_link=True, link_text='Export to plot.ly',
         figure_or_data, config, validate, '100%', 525, True
     )
 
-    ipython_display.display(ipython_display.HTML(plot_html))
+    figure = tools.return_figure_from_figure_or_data(figure_or_data, validate)
+
+    # Display 2 versions of the figure:
+    # - The raw HTML is the "old-school" style - we just throw in a JS script
+    #   inside the cell. This works for a lot of environments but some JS is
+    #   stripped out in some rendering environments / platforms like
+    #   GitHub, nteract, and eventually Plotly
```

Comment:
^^ I ended up just 🔪-ing a lot of that comment out, I don't think it needs to get committed here.